mapStyle.css

```diff
@@ -0,0 +1,50 @@
+/********************* maps - START */
+#map {
+    height: 60%;
+    width: 100%;
+}
+.controls {
+    margin-top: 10px;
+    border: 1px solid transparent;
+    border-radius: 2px 0 0 2px;
+    box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    height: 32px;
+    outline: none;
+    box-shadow: 0 2px 6px rgba(0, 0, 0, 0.3);
+}
+
+#pac-input {

```

Comment:
I'm assuming it means Places Auto Complete which is the search box embedded
on map. The map related code I got from Google Maps JavaScript API.

2016-03-02 17:23 GMT-03:00 André Luiz Cardoso <notifications@github.com>:

> In mapStyle.css
> <https://github.com/otaviofontana/arcTouchOtavioFontana/pull/1#discussion_r54784939>
> :
>
> > +#map {
> > +    height: 60%;
> > +    width: 100%;
> > +}
> > +.controls {
> > +    margin-top: 10px;
> > +    border: 1px solid transparent;
> > +    border-radius: 2px 0 0 2px;
> > +    box-sizing: border-box;
> > +    -moz-box-sizing: border-box;
> > +    height: 32px;
> > +    outline: none;
> > +    box-shadow: 0 2px 6px rgba(0, 0, 0, 0.3);
> > +}
> > +
> > +#pac-input {
>
> What pac stands for?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/otaviofontana/arcTouchOtavioFontana/pull/1/files#r54784939>
> .
>